src/.env.sample

```diff
@@ -147,5 +147,14 @@ RESOURCE_TYPES="projects,observations,observation_with_rubrics,surveys,programs"
 #maximum number of tasks allowed in a project
 MAX_PROJECT_TASK_COUNT= 10
 
+#consumption portal name ex: self/diksha/elevate-project
+CONSUMPTION_SERVICE=self
+
+#Publish method to consumption side kafka/api
+PUBLISH_METHOD=kafka
```

Comment:
@priyanka-TL we don't need this key, basically if API URL is given then push to API, if kafka topic is given we should push to Kafka, if both then do both.

Reply:
@aks30 if we publish to both method then duplicate template will create. So can we go publish through kafka or api ?